Sources/Testing/Issues/Issue.swift

```diff
@@ -103,6 +103,12 @@ public struct Issue: Sendable {
   /// The severity of this issue.
   @_spi(Experimental)
   public var severity: Severity
+  
+  /// If the issues is a failing issue.
```

Comment:
Can you use the documentation comment to explain what "failure" means in this context? Here's a very rough draft below. Before going with this, please make sure the double-backtick DocC symbol references actually resolve, since I just wrote them freehand.

```suggestion
  /// Whether or not this issue should cause the test it's associated with to be
  /// considered a failure.
  ///
  /// The value of this property is `true` for issues which have a severity level of
  /// ``Issue/Severity/error``` or greater and are not known issues via the ``withKnownIssue(_:...)``.
  /// Otherwise, the value of this property is `false.`
  ///
  /// Use this property to determine if an issue should be considered a failure, instead of
  /// directly comparing the value of the ``severity`` property.
```